web/src/components/UploadLicenseFile.jsx

```diff
@@ -36,22 +36,41 @@ class UploadLicenseFile extends React.Component {
 
   uploadLicenseFile = async () => {
     const { onUploadSuccess, history } = this.props;
-    const { licenseValue } = this.state;
+    const { licenseFile, licenseFileContent } = this.state;
 
     this.setState({
       fileUploading: true,
       errorMessage: "",
     });
+
+    let licenseText;
+    if (licenseFile.name.substr(licenseFile.name.lastIndexOf('.')) === ".rli") {
+      try {
+        // NOTE: this exchange is weird on the frontend but I don't wanna port
+        // all the uploadKotsLicense typescript code to go
```

Comment:
this is a little weird. i exchange the license via the frontend and make a second api call rather than doing it all on the api side of things. i guess i can rewrite the go code in typescript if i have to. i dont think i can port the typescript license upload to go.